.github/workflows/ci.yaml

```diff
@@ -34,7 +34,9 @@ jobs:
               run: |
                 pip install --upgrade pip
                 pip install tox
-                pip install mypy==0.800
+                pip install mypy==1.1.1
+                pip install types-PyYAML
+                pip install types-setuptools
```

Comment:
Completely naive question, is there no way to force the installation of those packages from .pre-commit-config.yaml?

Reply:
I think it was worth your time, as we will probably reuse this in Gitlab CIs too. So: thanks!